### ISSUE TITLE
vitest/valid-expect false errors on chai style expects

### ISSUE THREAD

**Issue by GNRSN** [veritem/eslint-plugin-vitest#237]
Hi, we're having a lot of mocha/chai test which we've migrated to vitest, written in chai syntax as is supported by vitest. The following example:
```
const fooBar = {foo: 'bar'};
expect(fooBar).to.deep.eq({foo: 'bar'});
```
Errors as `vitest/valid-expect`

I assume this is a bug/lacking support for chai syntax?

**Comment by veritem** [comment]
would you mind giving me a code repro i can play around with. That would help me fix it faster. Thanks.

**Comment by veritem** [comment]
what kind of error? would you mind sharing a repo with repro. I can fix this faster with more context. Thanks

**Comment by GNRSN** [comment]
The error description is "Expect has unknown modifier" (it's an eslint "Error") 

Thank you for your quick responses! 
I've created a repro here https://github.com/GNRSN/repro-vitest-eslint-plugin-237/tree/main

**Comment by yoavniran** [comment]
just moved to vitest and added this fine plugin. 
Getting the same problem (using chai matchers).

for example:

```
expect(value).to.eql("30");
```

error:
```
 6:3  error  Expect has unknown modifier  vitest/valid-expect
```

**Comment by RebeccaStevens** [comment]
This issue doesn't seem fixed.
